lissmilter.py

```diff
@@ -75,9 +77,14 @@ def envrcpt(self, mail_to, *str):
                 # Should only happen if no AssociationGroup called 'STYR' has been created yet
                 pass
 
+        # Check if sender is valid before discarding @studentspex.se addresses. A user sending 'from' @studentspex.se addresses should add this as an extra email in the database.
         if not valid_sender:
-            self.setreply('554', '5.7.2', 'Sender <{}> not authorized for recipient ("{}"). Kontakta en ansvarig om du tycker det borde funka.'.format(self.envelope_from, to))
-            return Milter.REJECT
+            if f'@{mail_domain}' in self.envelope_from:
```

Comment:
Det finns en stor risk att detta kommer ha sönder saker som skickar epost legitimt till en maillista från en @studentspex.se-adress, t.ex. nomineringssidan som skickar till valberedningen. 

Borde inte grundproblemet med denna typen av spam lösas om vi stramar åt SPF för vår domän från ~all som det är idag till -all så att vi får en hard fail istället för en soft fail?

Reply:
> Det finns en stor risk att detta kommer ha sönder saker som skickar epost legitimt till en maillista från en @studentspex.se-adress, t.ex. nomineringssidan som skickar till valberedningen.

Hm, det borde väl bara gälla om avsändaren inte räknas som giltig, och meddelandet tidigare förkastats med ett mailstuds?

Reply:
> Borde inte grundproblemet med denna typen av spam lösas om vi stramar åt SPF för vår domän från ~all som det är idag till -all så att vi får en hard fail istället för en soft fail?

Det är inte omöjligt (givet att vi inte då äter upp mail från lisse)! Det finns fler saker jag gärna kollar på vad gäller mailen och dess ursprungskontroll (både av studsskäl och annat). Den biten känner jag mig dock inte alls tillräckligt säker på för att vilja ge mig in och snabb-patcha. Här beskär vi bort ett specifikt beteende för en domän där vi bör kontrollera det beteendet, och fastslår huruvida icke-öppna spexmaillistor ska kunna få mailstuds.

Reply:
Eftersom jag har goda skäl att tro att ändrade TXT-records allena inte kommer att lösa detta (och nu även uttryckligt testat valberedningsfallet/klassen av fall), markerar jag den tråden som avklarad. Skapar ny issue kring detta, så att vi inte tappar bort det.